app/policies/book_policy.rb

```diff
@@ -0,0 +1,12 @@
+class BookPolicy < ApplicationController
+  attr_reader :user, :book
+
+  def initialize(user, book)
+    @user = user
+    @book = book
+  end
+
+  def create?
+    Rent.where(['"to" > ? AND book_id = ?', 'now()', book.id]).empty?
```

Comment:
Don't use strings in the where conditions, you can use where(id: book.id)

Reply:
I didnt find a better way to do it